Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -893,6 +899,16 @@ extension BrowserViewController: TabManagerDelegate {
 
         tab.webView.removeFromSuperview()
     }
+
+    func isValidURL(url: NSURL) -> Bool {
+        var validSchemes = ["about"]
+        if let scheme = url.scheme,
+            index = find(validSchemes, scheme) {
```

Comment:
Rather than listing schemes that are not web pages, how about we include only the schemes that are? In other words, return true if "http" or "https"; return false otherwise.